lib/ts/recipe/session/utils.ts

```diff
@@ -57,6 +57,17 @@ export async function sendUnauthorisedResponse(
     sendNon200Response(response, "unauthorised", recipeInstance.config.sessionExpiredStatusCode);
 }
 
+export async function sendMissingGrantResponse(
+    recipeInstance: SessionRecipe,
+    grantId: string,
+    __: BaseRequest,
+    response: BaseResponse
+) {
+    // TODO(grants): check header name
+    response.setHeader("missing-grant-id", grantId, false);
```

Comment:
- Will also need to modify the docs pertaining to this. The place that lists all the possible output.
- Any FDI spec update for this? -> it's probably a new FDI version then?

Reply:
Hmm, I'll check but I don't think we'd be setting this on any of our endpoints, only through verifySession